src/components/bootstrap-overrides.css

```diff
@@ -82,3 +85,16 @@
   line-height: 16px;
   color: #ffffff;
 }
+
+.story-block-text {
+font-family: Roboto, Helvetica, sans-serif;
+font-style: normal;
+font-weight: bold;
+/* or 133% */
+
+color: #707070;
```

Comment:
similarly, we want this color on the big-text section but _not_ on the other sections, at least according to the mock